README.md

```diff
@@ -568,9 +568,27 @@
 
 
 
-## Comparison Operators & Equality
+## Comparison Operators , Conditional (Ternary) Operators & Equality
```

Comment:
Please use an Oxford comma here in this list - `Comparison Operators, Conditional (Ternary) Operators, & Equality`